clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionService.java

```diff
@@ -1042,12 +1090,13 @@ public boolean killConsumptionTask(String topicName) {
 
```

Comment:
should we add a try-catch around entire kill method?

Reply:
Not aware of any existing issues. No need to be paranoid about unknown issues IMHO.